pkg/auth/acl/permission.go

```diff
@@ -0,0 +1,100 @@
+package acl
+
+import (
+	"fmt"
+
+	"github.com/treeverse/lakefs/pkg/auth"
+	"github.com/treeverse/lakefs/pkg/auth/model"
+	"github.com/treeverse/lakefs/pkg/permissions"
+)
+
+const (
+	// ACLRead allows reading the specified repositories, as well as
+	// managing own credentials.
+	ACLRead model.ACLPermission = "Read"
+	// ACLWrite allows reading and writing the specified repositories,
+	// as well as managing own credentials.
+	ACLWrite model.ACLPermission = "Write"
+	// ACLSuper allows reading, writing, and all other actions on the
+	// specified repositories, as well as managing own credentials.
+	ACLSuper model.ACLPermission = "Super"
+	// ACLAdmin allows all operations, including all reading, writing,
+	// and all other actions on all repositories, and managing
+	// authorization and credentials of all users.
+	ACLAdmin model.ACLPermission = "Admin"
+)
+
+var (
+	ACLPermissions = []model.ACLPermission{ACLRead, ACLWrite, ACLSuper, ACLAdmin}
+
+	ownUserARN = []string{permissions.UserArn("${user}")}
+	all        = []string{permissions.All}
+
+	ErrBadACLPermission = fmt.Errorf("%w: Bad ACL permission", model.ErrValidationError)
+)
+
+func RepositoriesToARNs(repositories model.Repositories) []string {
+	if repositories.All {
+		// "*" always treated globally, no specific ARN.
+		return all
+	}
+	arns := make([]string, len(repositories.List))
+	for i, repo := range repositories.List {
+		arns[i] = permissions.RepoArn(repo)
+	}
+	return arns
```

Comment:
This looks fine to me, but just in order to validate, 
`!repositories.All` => `len(repositories.List) !=0`
Is that the case? or is empty string a valid response?